package-lock.json

```diff
@@ -730,15 +730,14 @@
 			}
 		},
 		"ajv": {
-			"version": "5.5.2",
-			"resolved": "https://registry.npmjs.org/ajv/-/ajv-5.5.2.tgz",
-			"integrity": "sha1-c7Xuyj+rZT49P5Qis0GtQiBdyWU=",
-			"dev": true,
+			"version": "6.7.0",
+			"resolved": "https://registry.npmjs.org/ajv/-/ajv-6.7.0.tgz",
```

Comment:
im not sure this change is because of the added dependency in the `package.json` or it was meant to be updated 😕

Reply:
Corrected the commit